docs/UserGuide.md

```diff
@@ -4,33 +4,44 @@ title: User Guide
+  The purpose of this guide is to get you familiar with the features of **Jobby** - from the very basics, to the most advanced features the application has to offer. These features range from the simple task of adding an contacts into the application for tracking to how our application can assist you in tracking every steps of your application process. We will go through every feature **Jobby** has to offer within this guide. You can check out the Table of Contents to navigate to a feature you might be interested in using.
+
+  To begin using this guide, ensure you meet the [Minimum System Requirements](). Once this is done, go to the Quick Start section to get the app up and running. Once you have done so, please feel free to go through the features sections to find out what features Jobby has installed. We recommend going through it in the order of this guide but any order works as well! You may check out our table of contents to jump to any section of your choice.
 
 --------------------------------------------------------------------------------------------------------------------
 
+<div style="page-break-after: always"></div>
+## Table of contents
+* Table of Contents
+{:toc}
+<div style="page-break-after: always"></div>
+
 ## Installation
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have _Java 11_ or above installed in your computer.
+   * Download Java [here](https://www.oracle.com/java/technologies/downloads/)
+   * How to check your Java version on [Windows](https://www.howtogeek.com/717330/how-to-check-your-java-version-on-windows-10/) and [macOS](https://www.wikihow.com/Check-Java-Version-on-a-Mac) 
 
-1. Download the latest `jobby.jar` from [here](https://github.com/AY2324S1-CS2103T-W08-3/tp/releases).
+1. Download the latest _jobby.jar_ from [here](https://github.com/AY2324S1-CS2103T-W08-3/tp/releases).
 
 1. Copy the file to the folder you want to use as the _home folder_ for your Jobby Application.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar jobby.jar` command to run the application.<br>
-   A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+1. Open a command terminal, navigate into the folder you put the jar file in, and enter `java -jar jobby.jar` in the terminal to run the application.<br>
+   A window similar to the below should appear in a few seconds. Notice that the app contains some sample data.<br>
+   * If you do not know how to navigate to your folder in the terminal, check out these links for [Windows](https://www.howtogeek.com/659411/how-to-change-directories-in-command-prompt-on-windows-10/) and [macOS](https://appletoolbox.com/navigate-folders-using-the-mac-terminal/).
```

Comment:
Consider adding one for Linux

Reply:
Hmmmm for Linux I would expect them to be alr familiar with CLI.

Maybe I'll add that assumption inside.

Reply:
nvrmind ill add a section for Linux based systems